debian/control

```diff
@@ -18,6 +18,7 @@ Build-Depends: bison,
                libboost-regex-dev [amd64],
                libboost-system-dev [amd64],
                libboost-thread-dev [amd64],
+               libbz2-dev,
```

Comment:
@vuvova Did you decide that the MariaDB Server should from now on support all compression methods? I was under the impression that somebody was to do benchmarks and then select only a couple compression methods instead of having them all.

Reply:
Yes, because now they come "for free", they don't bring any extra dependencies, so we can enable them all, no trade-offs anymore.